components/CreateList/CreateListDrawer.tsx

```diff
@@ -63,7 +63,7 @@ export const CreateListDrawer = (): JSX.Element => {
           <Button onClick={onDrawerClose}>Add more resources</Button>
           <Button
             type="submit"
-            disabled={!isValid || createListItems.length === 0}
+            isDisabled={!isValid || createListItems.length === 0}
```

Comment:
[Prop changed](https://chakra-ui.com/docs/components/button/props) - I guess with the Chakra bump